test-data/unit/check-classes.test

```diff
@@ -3066,12 +3066,13 @@ class M(type):
 class A(metaclass=M): pass
 
 reveal_type(A[M])  # E: Revealed type is 'builtins.int'
+reveal_type(A[M])  # E: Revealed type is 'builtins.int'
```

Comment:
This reveal is identical to the previous one. I think you wanted to check something like ``A[A]``.